Quantum-Control-Applications/Superconducting/Single-Flux-Tunable-Transmon/configuration_with_lf_fem_and_mw_fem.py

```diff
@@ -0,0 +1,470 @@
+                    # Its range is -41dBm to +10dBm with 3dBm steps.
+                    "type": "MW",
+                    "analog_outputs": {
+                        1: {"band": 2, "full_scale_power_dbm": resonator_power},  # resonator
+                        2: {"band": 2, "full_scale_power_dbm": qubit_power},  # qubit
+                    },
+                    "digital_outputs": {},
+                    "analog_inputs": {
+                        1: {"band": 2},  # I from down-conversion
+                    },
+                },
+                lf_fem: {
+                    "type": "LF",
+                    "analog_outputs": {
+                        # Flux line
+                        1: {
+                            "offset": max_frequency_point,
+                            # The "output_mode" can be used to tailor the max voltage and frequency bandwidth, i.e.,
+                            #   "direct":    1Vpp (-0.5V to 0.5V), 750MHz bandwidth (default)
+                            #   "amplified": 5Vpp (-2.5V to 2.5V), 330MHz bandwidth
+                            "output_mode": "amplified",
+                            # The "sampling_rate" can be adjusted by using more FEM cores, i.e.,
+                            #   1 GS/s: uses one core per output (default)
+                            #   2 GS/s: uses two cores per output
+                            # NOTE: duration parameterization of arb. waveforms, sticky elements and chirping
+                            #       aren't yet supported in 2 GS/s.
+                            "sampling_rate": sampling_rate,
+                            # At 1 GS/s, use the "upsampling_mode" to optimize output for
+                            #   modulated pulses (optimized for modulated pulses):      "mw"    (default)
+                            #   unmodulated pulses (optimized for clean step response): "pulse"
+                            "upsampling_mode": "pulse",
+                        },
+                    },
+                    "digital_outputs": {
+                        1: {},
+                    },
+                },
+            },
+        },
+    },
+    "elements": {
+        "qubit": {
+            "MWInput": {
+                "port": (con, mw_fem, 1),
+                "oscillator_frequency": qubit_LO,
+            },
+            "intermediate_frequency": qubit_IF,
+            "operations": {
+                "cw": "const_pulse",
+                "saturation": "saturation_pulse",
+                "pi": "pi_pulse",
+                "pi_half": "pi_half_pulse",
+                "x180": "x180_pulse",
+                "x90": "x90_pulse",
+                "-x90": "-x90_pulse",
+                "y90": "y90_pulse",
+                "y180": "y180_pulse",
+                "-y90": "-y90_pulse",
+            },
+        },
+        "resonator": {
+            "MWInput": {
+                "port": (con, mw_fem, 2),
+                "oscillator_frequency": resonator_LO,
```

Comment:
As long as we're adding notes, let's add here (and for all outputs in all mw-fem configurations):
```
Note the 'oscillator_frequency' field will be renamed 'upconverter_frequency' and will be moved to the port definition in QOP 3.2. The ability to use multiple upconverters in the same output will also be added.
```

Reply:
Done